docs/mission_module.md

```diff
@@ -0,0 +1,118 @@
+# Supported devices
+
+The ONLY supported device is the Autonomy Device, serving as a driver for the car. The device is connected to the Mission Module and receives the mission to drive the car to the desired stops.
+
+This module handles backward compatibility with Fleet protocol v1.2.0. This version did not have modules and supported only one type of payload - CarStateProtocol, which contains statuses from autonomy and commands for autonomy. The payload for the module is serialized by protobuf v 3.21.12, the same version as the fleet protocol itself.
+
+Device ID currently has the following field values:
+- module ID: 1,
+- device type: 0,
+- device role: `driving`.
+
+The last field `device name` is not prescribed, as it is only informative and optional.
+
+# Autonomy Device
+
+The device drives the car according to the current mission (list of stops with names and position).
+
+## Messages
+
+### Structure
+
+The basic message structure for message in Internal and External protocol is described [here](https://ref.bringautofleet.com/r/protocol/v2/2.0.1/message-structure).
```

Comment:
This is not important. The module does not work wit internal structure. It known only its own data. It does not count or have any "impact" to how messages are delivered.
There needs to be only format of message which are produced/condumed by a module....